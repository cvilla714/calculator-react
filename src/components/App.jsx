/* eslint-disable quotes */
import React from "react";
import ButtonPanel from "./ButtonPanel";
import Display from "./Display";

function App() {
  return (
    <div>
      <h1 className="title">Project Calculator</h1>

      <div className="calculator">
        <Display result={0} />
        <ButtonPanel />
      </div>
    </div>
  );
}
export default App;
