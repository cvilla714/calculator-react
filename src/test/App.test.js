import React from 'react';
import renderer from 'react-test-renderer';
import App from '../components/App';

it('should render the correct content for the Calculator Page', () => {
  const tree = renderer.create(<App />).toJSON();
});
