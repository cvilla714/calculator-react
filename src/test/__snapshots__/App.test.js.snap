// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the correct content for the Calculator Page 1`] = `
<div
  className="container-fluid large"
>
  <div
    className="row"
  >
    <div
      className="col-sm-12 col-md-6"
    >
      <h1
        className="title mt-5 ml-5"
      >
        Let's do some math!
      </h1>
    </div>
    <div
      className="col-sm-12 col-md-6"
    >
      <div
        className="calculator mt-5"
      >
        <div
          className="display"
          id="display"
        >
          
          
          
          
        </div>
        <div
          className="group1"
        >
          <button
            className=" "
            onClick={[Function]}
            type="button"
          >
            AC
          </button>
          <button
            className=" "
            onClick={[Function]}
            type="button"
          >
            +/-
          </button>
          <button
            className=" "
            onClick={[Function]}
            type="button"
          >
            %
          </button>
          <button
            className="orange "
            onClick={[Function]}
            type="button"
          >
            รท
          </button>
          <button
            className=" "
            onClick={[Function]}
            type="button"
          >
            7
          </button>
          <button
            className=" "
            onClick={[Function]}
            type="button"
          >
            8
          </button>
          <button
            className=" "
            onClick={[Function]}
            type="button"
          >
            9
          </button>
          <button
            className="orange "
            onClick={[Function]}
            type="button"
          >
            *
          </button>
          <button
            className=" "
            onClick={[Function]}
            type="button"
          >
            4
          </button>
          <button
            className=" "
            onClick={[Function]}
            type="button"
          >
            5
          </button>
          <button
            className=" "
            onClick={[Function]}
            type="button"
          >
            6
          </button>
          <button
            className="orange "
            onClick={[Function]}
            type="button"
          >
            -
          </button>
          <button
            className=" "
            onClick={[Function]}
            type="button"
          >
            1
          </button>
          <button
            className=" "
            onClick={[Function]}
            type="button"
          >
            2
          </button>
          <button
            className=" "
            onClick={[Function]}
            type="button"
          >
            3
          </button>
          <button
            className="orange "
            onClick={[Function]}
            type="button"
          >
            +
          </button>
          <button
            className=" bigN"
            onClick={[Function]}
            type="button"
          >
            0
          </button>
          <button
            className=" "
            onClick={[Function]}
            type="button"
          >
            .
          </button>
          <button
            className="orange "
            onClick={[Function]}
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
